lib/Gocdb_Services/Config.php

```diff
@@ -377,6 +377,22 @@ public function getReservedScopeList() {
         }
         return $reservedScopes;
     }
+    public function getFilterDowntimesByScope () {
```

Comment:
why the special case for downtimes?

Reply:
Not sure exactly what you ask, and it's been a long time since, but AFAIR discussions, the display/filtering of downtimes was inconsistent compared to other entities in being filtered by default. Introducing the new config flag filter_downtimes_by_default_scope especially for downtimes allows, if set to "true", to mimic existing behaviour. If set to false, downtimes, along with everything else, are only filtered if the default_filter_by_scope flag is true.

Reply:
following some investigation, @ineilson and I had trouble recreating the existing behaviour for both the `Downtimes_Calendar` page and the `Sites` page .

it seems as if the two filtering options (`default_filter_by_scope` and `filter_downtimes_by_default_scope`) may be linked in some way (if one is false the other is always false perhaps) and I dont think they should be.

Reply:
Summary of offline discussions: decided to treat the current Downtimes_Calendar page behaviour (i.e. silently filtering by `default scope`) as a bug and make all pages respond consistently to the setting of `default_filter_by_scope`, eliminating the need for `filter_downtimes_by_default_scope`.

Reply:
`filter_downtimes_by_default_scope` removed in 1b51b69234b9827a75535fc6fdd67d5a9c1006a9